app/test/api/meldekortperioder/MeldekortPeriodeTest.kt

```diff
@@ -1,86 +1,49 @@
 import io.ktor.client.request.*
 import io.ktor.http.*
-import io.ktor.serialization.jackson.*
-import io.ktor.server.testing.*
+import kotlinx.coroutines.runBlocking
 import no.nav.aap.arenaoppslag.kontrakt.intern.InternVedtakRequest
-import no.nav.aap.komponenter.dbtest.InitTestDatabase
 import no.nav.aap.komponenter.type.Periode
 import org.junit.jupiter.api.Assertions.assertEquals
 import org.junit.jupiter.api.Test
 import java.time.LocalDate
 
-private val dataSource2 = InitTestDatabase.freshDatabase()
-
-class MeldekortPeriodeTest : PostgresTestBase(dataSource2) {
+class MeldekortPeriodeTest : TestBase() {
 
     @Test
     fun `kan lagre ned og hente meldekort perioder`() {
-        Fakes().use { fakes ->
-            val config = TestConfig.default(fakes)
-            val azure = AzureTokenGen("test", "test")
-
-            testApplication {
-                application {
-                    api(
-                        config = config,
-                        datasource = dataSource2,
-                        arenaRestClient = ArenaClient()
+        runBlocking {
+            val perioder = listOf(
+                Periode(LocalDate.ofYearDay(2021, 1), LocalDate.ofYearDay(2021, 15)),
+                Periode(LocalDate.ofYearDay(2021, 16), LocalDate.ofYearDay(2021, 31))
+            )
+
+            val res = httpClient.post("/api/insert/meldeperioder") {
+                bearerAuth(azure.generate(true))
+                contentType(ContentType.Application.Json)
+                setBody(
+                    MeldekortPerioderDTO(
+                        "12345678910",
+                        perioder
                     )
-                }
-
-                val perioder = listOf(
-                    Periode(LocalDate.ofYearDay(2021, 1), LocalDate.ofYearDay(2021, 15)),
-                    Periode(LocalDate.ofYearDay(2021, 16), LocalDate.ofYearDay(2021, 31))
                 )
+            }
 
-                val res = jsonHttpClient.post("/api/insert/meldeperioder") {
-                    bearerAuth(azure.generate(true))
-                    contentType(ContentType.Application.Json)
-                    setBody(
-                        MeldekortPerioderDTO(
-                            "12345678910",
-                            perioder
-                        )
-                    )
-                }
-
-                assertEquals(HttpStatusCode.OK, res.status)
-                assertEquals(countMeldekortEntries(), 2)
-
-                val meldekortPerioderRes = jsonHttpClient.post("/perioder/meldekort") {
-                    bearerAuth(azure.generate(true))
-                    contentType(ContentType.Application.Json)
-                    setBody(InternVedtakRequest("12345678910", LocalDate.ofYearDay(2021, 1), LocalDate.ofYearDay(2021, 31)))
-                }
+            assertEquals(HttpStatusCode.OK, res.status)
+            assertEquals(countMeldekortEntries(), 2)
```

Comment:
Hmm, nå kjører vel alle testene i samme database? I behandlingsflyt har vi pleid å lage ny transaksjon for hver test. Og kalle freshDatabase(). Går jo bra så lenge man unngår å gjenbruke UUID-er, så sikkert ikke så farlig